spring/src/main/java/io/cucumber/spring/FixBootstrapUtils.java

```diff
@@ -57,7 +52,7 @@
 			"org.springframework.test.context.web.WebAppConfiguration";
 
     static BootstrapContext createBootstrapContext(Class<?> testClass) {
-        CacheAwareContextLoaderDelegate contextLoader = new DefaultCacheAwareContextLoaderDelegate(contextCache.get());
+        CacheAwareContextLoaderDelegate contextLoader = new DefaultCacheAwareContextLoaderDelegate();
```

Comment:
Not passing in a context will use a static (thread safe) context cache, which will be shared between threads